paparazzi/paparazzi-gradle-plugin/src/test/java/app/cash/paparazzi/gradle/PaparazziPluginTest.kt

```diff
@@ -191,6 +191,18 @@ class PaparazziPluginTest {
     assertThat(snapshotsDir.exists()).isFalse()
   }
 
+  @Test
+  fun buildClassNextSdkAccess() {
+    val fixtureRoot = File("src/test/projects/build-class-next-sdk")
```

Comment:
Happy to take alternate suggestions for the naming of `build-class-next-sdk` that will be the most clear for future maintenance.